liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/server/RunServerMojo.java

```diff
@@ -24,7 +24,7 @@
 /**
  * Start a liberty server
  */
-@Mojo(name = "run", requiresDependencyCollection = ResolutionScope.COMPILE, requiresDependencyResolution = ResolutionScope.COMPILE)
+@Mojo(name = "run", requiresDependencyResolution = ResolutionScope.COMPILE_PLUS_RUNTIME)
```

Comment:
Why was `requiresDependencyCollection` removed?

Reply:
I looked at what the deploy goal had and it did not have that. I also looked up the documentation for it, and thought it was not necessary. I removed it and all tests passed. So I presumed I was correct. But I am open to an explanation on why it is needed.

Reply:
I'm not sure what the difference is, and the documentation doesn't provide any additional information. Perhaps a dependency must be collected before it can be resolved? This answer states that both are important: https://stackoverflow.com/a/42559904/634324

It is probably better to just keep both, to be safe.